operator/pkg/helmreconciler/apply.go

```diff
@@ -230,3 +251,72 @@ func (h *HelmReconciler) serverSideApply(obj *unstructured.Unstructured) error {
 	}
 	return nil
 }
+
+func (h *HelmReconciler) analyzeWebhooks(whs object.K8sObjects) error {
+	// Get YAML string from K8sObject
+	var whYAMLs []string
+	whNameMap := make(map[string]bool)
+	for _, whObj := range whs {
+		yaml, err := whObj.YAML()
+		if err != nil {
+			return err
+		}
+		whYAMLs = append(whYAMLs, string(yaml))
+		whNameMap[whObj.Name] = true
+	}
+	if len(whYAMLs) == 0 {
+		return nil
+	}
+
+	globalI, ok := h.iop.Spec.Values["global"]
+	if !ok {
+		return fmt.Errorf("")
```

Comment:
```suggestion
		return fmt.Errorf("failed to get global value")
```

Should have some useful error aside from `""`. There are a few other locations as well

Reply:
Yes definitely, Since it's a draft PR I didn't add anything yet

Reply:
Added error messages